src/Entity/Property/EmailPropertyAwareTrait.php

```diff
@@ -24,18 +24,36 @@ trait EmailPropertyAwareTrait
     protected $email;
```

Comment:
Is this trait used anywhere else?

Reply:
Yes, this is a generic trait for setting/getting email address.

Reply:
Does it need it's own trail is what I am thinking. OriginalEmailPropertyAwareTrait ?

so if someone wanted to use EmailPropertyAwareTrait (e.g. DeveloperApps) trait in the future they would get this property by default even if they didn't want it.

Reply:
@giteshk I removed it from `EmailPropertyAwareTrait` and implemented in `Developer` entity. [Here](https://github.com/apigee/apigee-client-php/pull/154/commits/7c7a4fe24cd89cdac34836ba97813c171584bb6c)
(It is a method just to save the Original email address so trait required.)